src/controller/CHIPDeviceController.cpp

```diff
@@ -1830,6 +1832,27 @@ void DeviceCommissioner::AdvanceCommissioningStage(CHIP_ERROR err)
     mCommissioningStage = nextStage;
 }
 
+CHIP_ERROR DeviceCommissioner::GetRandomOperationalNodeId(NodeId * aNodeId)
```

Comment:
This should not be part of Commissioner, but a utility in `src/controller/ExampleOperationalCredentialsIssuer.cpp` or similar. While in some examples the commissioner may allocate NodeIDs, the common way will be from the CA.

Reply:
That was my point here as well: https://github.com/project-chip/connectedhomeip/pull/8450#discussion_r671416311.